introduction/bloco_8/dia_1/script.js

```diff
@@ -0,0 +1,35 @@
+// Exercise 1
+const functionFirstOrder = (fullName) => {
+  return {
+    fullName,
+    email: fullName.toLowerCase().split(' ').join('_')+`@trybe.com`,
+  };
+}
+const newEmployees = (superFunction) => {
+  const employees = {
+    // Nome: Pedro Guerra -> Chame sua função passando o nome Pedro Guerra como parâmetro
+    id1: superFunction('Pedro Guerra'),
+    // Nome: Luiza Drumond -> Chame sua função passando o nome Luiza Drumond como parâmetro
+    id2: superFunction('Luiza Drumond'),
+    // Nome: Carla Paiva -> Chame sua função passando o nome Carla Paiva como parâmetro
+    id3: superFunction('Carla Paiva') 
+  }
+  return employees;
+};
+
+console.log(newEmployees(functionFirstOrder));
+
+// Exercise 2
+const sorteio = () => {
+  const numberRandom = Math.floor((Math.random() * 5) + 1);
```

Comment:
Ao invés de usar o `Math.floor()` e adicionar 1, você pode usar o `Math.ceil()` que ele já arredonda pra cima!
```suggestion
  const numberRandom = Math.ceil(Math.random() * 5);
```